### PR TITLE
Add deploy to heroku button

### PR DESCRIPTION
**Pull request by Tama63**
Makes hosting the API easier for others.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Inspired by [Gravatar](https://gravatar.com) (hence the name) and [Minotar](http
 
 Image manipulation is done by [lwip](https://github.com/EyalAr/lwip). Renders are created with [node-canvas](https://github.com/Automattic/node-canvas), based on math by [confuser](https://github.com/confuser/serverless-mc-skin-viewer).
 
-![redstone_sheep's avatar](https://crafatar.com/avatars/ae795aa86327408e92ab25c8a59f3ba1?size=128) ![Jake0oo0's avatar](https://crafatar.com/avatars/2d5aa9cdaeb049189930461fc9b91cc5?size=128) ![Notch's avatar](https://crafatar.com/avatars/069a79f444e94726a5befca90e38aaf5?size=128) ![sk89q's avatar](https://crafatar.com/avatars/0ea8eca3dbf647cc9d1ac64551ca975c?size=128) ![md_5's avatar](https://crafatar.com/avatars/af74a02d19cb445bb07f6866a861f783?size=128) 
+![redstone_sheep's avatar](https://crafatar.com/avatars/ae795aa86327408e92ab25c8a59f3ba1?size=128) ![Jake0oo0's avatar](https://crafatar.com/avatars/2d5aa9cdaeb049189930461fc9b91cc5?size=128) ![Notch's avatar](https://crafatar.com/avatars/069a79f444e94726a5befca90e38aaf5?size=128) ![sk89q's avatar](https://crafatar.com/avatars/0ea8eca3dbf647cc9d1ac64551ca975c?size=128) ![md_5's avatar](https://crafatar.com/avatars/af74a02d19cb445bb07f6866a861f783?size=128)
```
Comment (jomo) [inline README.md]: pro tip: although this change was reasonable, removing [trailing spaces in markdown](http://daringfireball.net/projects/markdown/syntax#p) isn't always a good idea.
> When you do want to insert a \<br /\> break tag using Markdown, you end a line with two or more spaces, then type return.

**Comment by jomo** [comment]
Can you move the button down to the `Install` section? doesn't really line up nicely with the status things

File: README.md (hunk 1 above)
Comment (Tama63) [inline README.md]: Sorry about this seems like Atom's doing

**Comment by Tama63** [comment]
@jomo Done and pushing now

**Comment by jomo** [comment]
thanks!

**Pull request merged**